### PR TITLE
Firebase update

### PR DESCRIPTION
**Pull request by chcx97**

### CODE REVIEW

**Review by annesatran — APPROVED** [review]

File: balto-ui/src/components/AddDogRecord/AddDogRecord.jsx [hunk 1]
```diff
@@ -95,13 +126,23 @@ export default function AddDogRecord() {
...
-
+  React.useEffect(()=>{
+    if (isLoading) {
+      simulateNetworkRequest().then(() => {
+        setLoading(false);
+        setShow(true)
+      });
+    }
+    if (form.imageUrl && form.imageName){
+      console.log("image saved!")
```
Comment (Rebecca-CYLi) [inline balto-ui/src/components/AddDogRecord/AddDogRecord.jsx]: found a console log

File: balto-ui/src/components/EditDogRecord/EditDogRecord.jsx [hunk 2]
```diff
@@ -42,7 +53,40 @@ export default function EditDogRecord() {
...
   }
+  const handleOnImageFileChange = (evt) => {
+    if (evt.target.files[0]){
+      setImageUpload(evt.target.files[0])
+    }
+  }
+  function simulateNetworkRequest() {
+    return new Promise((resolve) => setTimeout(resolve, 2000));
+  }
+  console.log(form.imageUrl)
```
Comment (Rebecca-CYLi) [inline balto-ui/src/components/EditDogRecord/EditDogRecord.jsx]: found a console log

File: balto-ui/src/components/EditDogRecord/EditDogRecord.jsx [hunk 3]
```diff
@@ -103,7 +147,19 @@ export default function EditDogRecord() {
       navigate("/admin-dashboard/dog-record/id/"+dogId)
     }
   }
-
+  console.log(form.image_url)
```
Comment (Rebecca-CYLi) [inline balto-ui/src/components/EditDogRecord/EditDogRecord.jsx]: found a console log

File: balto-ui/src/components/EditDogRecord/EditDogRecord.jsx [hunk 4]
```diff
@@ -103,7 +147,19 @@ export default function EditDogRecord() {
...
+  React.useEffect(()=>{
+    if (isLoading) {
+      simulateNetworkRequest().then(() => {
+        setLoading(false);
+        setShow(true);
+      });
+    }
+    if (form.image_url && form.image_name){
+      console.log("image saved!")
```
Comment (Rebecca-CYLi) [inline balto-ui/src/components/EditDogRecord/EditDogRecord.jsx]: found a console log

File: balto-ui/src/components/EditDogRecord/EditDogRecord.jsx [hunk 5]
```diff
@@ -103,7 +147,19 @@ export default function EditDogRecord() {
...
+    }
+  }, [form.image_url, form.image_name, isLoading])
+  console.log(dogRecord.image_name, dogRecord.image_url)
```
Comment (Rebecca-CYLi) [inline balto-ui/src/components/EditDogRecord/EditDogRecord.jsx]: found a console log

**Review by Rebecca-CYLi — CHANGES_REQUESTED** [review]
found console logs, please fix them

**Review by Rebecca-CYLi — APPROVED** [review]
console logs removed, G2G

**Pull request merged**